src/GT-Debugger/GTSpecPreDebugWindow.class.st

```diff
@@ -179,26 +163,24 @@ GTSpecPreDebugWindow >> initializeCodePane [
 
 { #category : #initialization }
 GTSpecPreDebugWindow >> initializePresenter [
-
 	super initializePresenter.
-	
-	debugger whenChangedDo: [ :aDebugger |
-		aDebugger 
-			ifNil: [ self clearWidget ] 
-			ifNotNil:  [ self rebuildWidget ]	 ]
 
+	debugger whenChangedDo: [ :aDebugger | aDebugger ifNotNil: [ self rebuildWidget ] ]
 ]
 
 { #category : #'initialization widgets' }
 GTSpecPreDebugWindow >> initializeStackPane [
 	self stackPane
-		displayBlock: [ :aContext | self columnsFor: aContext ];
+		addColumn: (StringTableColumn title: 'Class' evaluated: [ :aContext | self debugger formatStackClassColumnForContext: aContext ]);
```

Comment:
this will have a problem because this methods answer a Text and they will be editable in new Table Presenter

Reply:
Is there a kind of column managing text or should we create it/modify the string one?